src/App.js

```diff
@@ -9,7 +11,9 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
Враховуючи, що в тебе 3 параметри з присвоєнням, то краще переносити все на окремий рядок
```suggestion
    <Article 
      title={article.title} 
      date={article.date} 
      text={article.text} 
    />
```

Reply:
Приймається! Дякую!